docs/guides/integration-prebid-client-side.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Prebid.js Client-Side Integration
+sidebar_label: Prebid.js Client-Side Integration
+pagination_label: Prebid.js Client-Side Integration
+description: Information about setting up a client-side Prebid.js integration.
+hide_table_of_contents: false
+sidebar_position: 04
+---
+
+# Prebid.js Client-Side Integration
+
+This guide is for publishers who have access to [DII](../ref-info/glossary-uid.md#gl-dii) (email address or phone number) on the client side and want to integrate with UID2 and generate [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) (advertising tokens) to be passed by Prebid.js in the RTB bid stream.
+
+To integrate with UID2 using Prebid.js, you'll need to make changes to the HTML and JavaScript on your site. No server-side work is required if you follow this guide.
+
+<!-- 
+This guide includes the following information:
+
+- [Prebid.js Version](#prebidjs-version)
+- [Integration Example](#integration-example)
+- [Integration Overview: High-Level Steps](#integration-overview-high-level-steps)
+   - [Complete UID2 Account Setup](#complete-uid2-account-setup)
+   - [Add Prebid.js to Your Site](#add-prebidjs-to-your-site)
+   - [Configure the UID2 Module](#configure-the-uid2-module)
+- [Storing the UID2 Token in the Browser](#storing-the-uid2-token-in-the-browser)
+- [When to Pass DII to the UID2 Module](#when-to-pass-dii-to-the-uid2-module)
+- [Checking the Integration](#checking-the-integration)
+- [Optional: Reduce Latency by Setting the API Base URL](#optional-reduce-latency-by-setting-the-api-base-url)
+ -->
+
+## Prebid.js Version
+
+This implementation requires Prebid.js version 8.21.0 or later. For version information, see [https://github.com/prebid/Prebid.js/releases](https://github.com/prebid/Prebid.js/releases).
+
+If you need to use an earlier version of Prebid.js, use the implementation solution presented in [Prebid.js Server-Side Integration](integration-prebid-server-side.md) instead.
+
+## Module Storage
```

Comment:
i just noticed we have the _Storing the UID2 Token in the Browser_ section below  which overlaps this section. We can remove Module Storage here and same in server side guide

Reply:
Good catch, thx, done.